app/controllers/employees/rewards_controller.rb

```diff
@@ -1,7 +1,7 @@
 module Employees
   class RewardsController < EmployeesController
     def index
-      render :index, locals: { rewards: Reward.all }
+      @rewards = Reward.all
```

Comment:
Why did you decide to switch from local variables to instance variables?

Reply:
I honestly don't remember. It had to do something with my experiments with the code. At some point i thought that only instance var will work. Nevertheless you'll be pleased that (since order works only in the back-end) i got rid of all instance vars there also.